contracts/interfaces/OtokenInterface.sol

```diff
@@ -2,17 +2,17 @@
 pragma solidity 0.6.10;
 
 interface OtokenInterface {
-    function underlyingAsset() external returns (address);
+    function underlyingAsset() external view returns (address);
```

Comment:
can we add `get` to the functions names, so it is clear that those functions are getters?

Reply:
it's not actually a getter function, just the public variables we declared in the Otoken contract.